cfg/config.go

```diff
@@ -64,37 +68,37 @@ func GetConfigAndValidate() (*Config, error) {
 		config.Parallelism = 4
 	}
 
-	bufferSize := os.Getenv("ED_BUFFER_SIZE")
+	bufferSize := os.Getenv("ED_BUFFER_SIZE_IN_BYTES")
 	if bufferSize != "" {
 		if i, err := strconv.ParseInt(bufferSize, 10, 0); err == nil {
 			config.BufferSize = int(i)
 		} else {
-			multiErr = append(multiErr, fmt.Sprintf("Unable to parse BUFFER_SIZE: %v", err))
+			multiErr = append(multiErr, fmt.Sprintf("Unable to parse BUFFER_SIZE_IN_BYTES: %v", err))
 		}
 	} else {
-		config.BufferSize = 100
+		config.BufferSize = 10 * 1000 * 1000
 	}
 
-	retryTimeout := os.Getenv("ED_RETRY_TIMEOUT")
-	if retryTimeout != "" {
-		if i, err := strconv.ParseInt(retryTimeout, 10, 0); err == nil {
-			config.RetryTimeout = time.Duration(i)
+	pushTimeout := os.Getenv("ED_PUSH_TIMEOUT")
```

Comment:
we need to update existing lambdas deployed with this extension and also update the documentation.